service/src/main/java/com/ericsson/eiffel/remrem/generate/constants/RemremGenerateServiceConstants.java

```diff
@@ -35,6 +35,9 @@
         public static final String UNAVAILABLE_FOR_FAILIFNONEFOUND = "{\"status_code\": 406, \"result\": \"FAIL\", "
                 + "\"message\":\"No event id found with ERLookup properties\"}";
 
+        public static final String UNAVAILABLE_LOOKUP_OPTIONS = "{\"status_code\": 422, \"result\": \"FAIL\", "
+                + "\"message\":\"Link specific options could not fetch information from ER\"}";
```

Comment:
I would prefer an error message like 'Link specific lookup options could not be fulfilled'. And I guess the string could be more generically called 'LOOKUP_OPTIONS_NOT_FULFILLED' instead of 'UNAVAILABLE_LOOKUP_OPTIONS'

Reply:
changed error message and variable name as suggested.

Reply:
Thanks